src/plugins/intel_cpu/tests/functional/test_utils/x64/filter_cpu_info.cpp

```diff
@@ -0,0 +1,63 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "test_utils/cpu_test_utils.hpp"
+#include "test_utils/filter_cpu_info.hpp"
+#include "ie_ngraph_utils.hpp"
+#include "openvino/core/type/element_type.hpp"
+#include "utils/rt_info/memory_formats_attribute.hpp"
+#include "utils/general_utils.h"
+#include <cstdint>
+
+namespace CPUTestUtils {
+
+std::vector<CPUSpecificParams> filterCPUInfo(const std::vector<CPUSpecificParams>& CPUParams) {
+    std::vector<CPUSpecificParams> archCPUParams = filterCPUInfoForArch(CPUParams);
+    std::vector<CPUSpecificParams> deviceCPUParams = filterCPUInfoForDevice(archCPUParams);
+    return deviceCPUParams;
+}
+
+std::vector<CPUSpecificParams> filterCPUInfoForArch(const std::vector<CPUSpecificParams>& CPUParams) {
+    std::vector<CPUSpecificParams> resCPUParams;
+    const int selectedTypeIndex = 3;
+
+    for (auto param : CPUParams) {
+        auto selectedTypeStr = std::get<selectedTypeIndex>(param);
+
+        if (selectedTypeStr.find("acl") != std::string::npos)
```

Comment:
I think we need to have "allow" list instead of "disallow".
So, we don't need to update this code when the new backends/implementations are introduced.
But we need to have better naming for the implementations then.
This will be partially done in scope of executors refactoring.
So, I am personally ok to stick with the current implementation for now.